projects/schematics/src/shared/utils/lib-utils.ts

```diff
@@ -180,9 +216,15 @@ export function shouldAddFeature(
 
 export function addLibraryFeature<T extends LibraryOptions>(
   options: T,
-  config: FeatureConfig
+  config: SchematicConfig
 ): Rule {
   return (tree: Tree, context: SchematicContext) => {
+    if (options.debug) {
+      context.logger.info(
```

Comment:
let's make use of your debugLog()

Reply:
Done. I renamed the `debugLog` to `debugLogRule`